### PR TITLE
[zh]translate docs:migration-from-kubefed.md

### PR DESCRIPTION
**Pull request by justedennnnn**
**What type of PR is this?**

/kind documentation

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 1]
```diff
@@ -178,11 +181,11 @@ spec:
         - cluster2
 ```
 
-The `PropagationPolicy` defines the rules of which resources(`resourceSelectors`) should be propagated to 
-where (`placement`). 
-See [Resource Propagating](https://karmada.io/docs/userguide/scheduling/resource-propagating) for more details.
+`PropagationPolicy` 定义了哪些资源 (`resourceSelectors`) 应该被传播到 在哪里 (`placement`)。
+参见[资源传播](https://karmada.io/docs/userguide/scheduling/resource-propagating)了解更多细节。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:185]:
```suggestion
参见[资源调度](https://karmada.io/docs/userguide/scheduling/resource-propagating)了解更多细节。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 2]
```diff
@@ -178,11 +181,11 @@ spec:
...
+
+对于 `override` 部分，Karmada 提供了 `OverridePolicy`  API 来保存区分的规则：
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:187]:
```suggestion
对于 `override` 部分，Karmada 提供了 `OverridePolicy` API 来保存差异化规则：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 3]
```diff
@@ -215,19 +218,16 @@ spec:
             value: 1.17.0-alpine
 ```
 
-The `OverridePolicy` defines the rules of which resources(`resourceSelectors`) should be overwritten when 
-propagating to where(`targetCluster`).
+`OverridePolicy` 定义了哪些资源 (`resourceSelectors`) 在传播到 where（`targetCluster`）时应该被覆盖的规则传播到哪里（`targetCluster`）。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:221]:
```suggestion
`OverridePolicy` 定义了哪些资源 (`resourceSelectors`) 在调度到哪里 (`targetCluster`) 时应该被覆盖的规则。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 4]
```diff
@@ -215,19 +218,16 @@ spec:
...
 
-In addition to Kubefed, Karmada offers various alternatives to declare the override rules, see
-[Overriders](https://karmada.io/docs/userguide/scheduling/override-policy#overriders) for more details.
+除了 Kubefed 之外，Karmada 还提供了各种替代方案来声明覆盖规则，见[Overriders](https://karmada.io/docs/userguide/scheduling/override-policy#overriders)了解更多细节。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:223]:
```suggestion
除了 Kubefed 之外，Karmada 还提供了各种替代方案来声明差异化规则，参见
[Overrider](https://karmada.io/docs/userguide/scheduling/override-policy#overriders)了解更多细节。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 5]
```diff
@@ -215,19 +218,16 @@ spec:
...
 
 ## FAQ
 
-### Will Karmada provide tools to smooth the migration?
+### Karmada 会提供平滑迁移的工具吗？
 
-We don't have the plan yet, as we reached some Kubefed users and found that they're usually not using vanilla 
-Kubefed but the forked version, they extended Kubefed a lot to meet their requirements. So, it might be pretty
-hard to maintain a common tool to satisfy most users.
+我们还没有这个计划，因为我们接触了一些 Kubefed 用户，发现他们通常不使用 Vanilla Kubefed，而是分叉版本。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:229]:
```suggestion
我们还没有这个计划，因为我们接触了一些 Kubefed 用户，发现他们通常不使用 Vanilla Kubefed，而是使用克隆版本。
```

**Review by windsonsea — COMMENTED** [review]
Some terms like `override` and `propagation` will be clear if you can look up through the documents. Comments here are only for reference. Thanks.

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 6]
```diff
@@ -1,54 +1,50 @@
 ---
-title: Migration From Kubefed
+title: 从 Kubefed 迁移
 ---
 
-Karmada is developed in continuation of Kubernetes [Federation v1](https://github.com/kubernetes-retired/federation) 
-and [Federation v2(aka Kubefed)](https://github.com/kubernetes-sigs/kubefed). Karmada inherited a lot of concepts
-from these two versions. For example:
+Karmada 是在 Kubernetes [Federation V1](https://github.com/kubernetes-retired/federation)中开发的
+和[Federation V2(又名 Kubefed)](https://github.com/kubernetes-sigs/kubefed)。Karmada 继承了很多概念
```
Comment (SAMZONG) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:6]:
```suggestion
Karmada 是在 Kubernetes [Federation V1](https://github.com/kubernetes-retired/federation)
和[Federation V2(又名 Kubefed)](https://github.com/kubernetes-sigs/kubefed)上开发的。Karmada 继承了很多两者的概念
```

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 7]
```diff
@@ -1,54 +1,50 @@
...
+来自这两个版本。例如：
 
-- **Resource template**: Karmada uses Kubernetes Native API definition for federated resource template, 
-  to make it easy to integrate with existing tools that already adopt Kubernetes.
-- **Propagation Policy**: Karmada offers a standalone Propagation(placement) Policy API to define multi-cluster 
-  scheduling and spreading requirements.
-- **Override Policy**: Karmada provides a standalone Override Policy API for specializing cluster relevant
-  configuration automation. 
+- **资源模板**：Karmada 使用 kubernetes 本机 API 定义用于联合资源模板，使其易于与已经采用 Kubernetes 的现有工具集成。
+- **繁殖政策**：Karmada 提供独立的传播（位置）策略 API 来定义多集群调度和分布要求。
```
Comment (SAMZONG) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:10]:
```suggestion
- **传播策略**：Karmada 提供独立的传播（位置）策略 API 来定义多集群调度和分布要求。
```
一般情况下，我们将 `Policy` 翻译为 策略，Propagation Policy 更合适称之为 传播策略， 你可以在很多 Karmada 的文章找到这部分.

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 8]
```diff
@@ -215,19 +218,16 @@ spec:
...
-Kubefed but the forked version, they extended Kubefed a lot to meet their requirements. So, it might be pretty
-hard to maintain a common tool to satisfy most users.
+我们还没有这个计划，因为我们接触了一些 Kubefed 用户，发现他们通常不使用 Vanilla Kubefed，而是分叉版本。
+他们对 Kubefed 进行了大量扩展以满足他们的需求。
+所以，要维护一个通用的工具来满足大多数用户的需求可能是相当困难的。
 
-We are also looking forward more feedback now, please feel free to reach us, and we are glad to support you
-finish the migration.
\ No newline at end of file
+我们现在也在期待更多的反馈，请随时联系我们，我们很乐意支持您完成迁移工作。
```
Comment (SAMZONG) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:233]:
```suggestion
我们非常期待更多的反馈，请随时与我们联系，我们将乐意支持您完成迁移工作。
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
汗，这大部分都没改呀，怎么就点了 Resolve conversation 了？

**Comment by RainbowMango** [comment]
@justedennnnn Can you help to resolve the conflicts, the typo in `docs/administrator/migration/migration-from-kubefed.md` has been resolved in #416.

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md (hunk 7 above)
Comment (RainbowMango) [reply i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:10]: 这里还是翻译成 `分发策略`吧。

Comment (Poor12) [reply i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:10]: Propagation Policy 最好不要翻译成传播策略，传播这个词更多用于新闻与传媒相关的领域。如果在其他Karmada的文章中找到，我们最好一起修正下。

**Review by Poor12 — COMMENTED** [review]
Please check again whether the previous review comments have been modified.

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md (hunk 7 above)
Comment (justedennnnn) [reply i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:10]: ok

**Comment by justedennnnn** [comment]
> /hold until all comments are really resolved

sorry，forget to save changes

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 9]
```diff
@@ -1,54 +1,50 @@
...
+Karmada 继承了很多两者的概念。例如：
 
-- **Resource template**: Karmada uses Kubernetes Native API definition for federated resource template, 
-  to make it easy to integrate with existing tools that already adopt Kubernetes.
-- **Propagation Policy**: Karmada offers a standalone Propagation(placement) Policy API to define multi-cluster 
-  scheduling and spreading requirements.
-- **Override Policy**: Karmada provides a standalone Override Policy API for specializing cluster relevant
-  configuration automation. 
+- **资源模版**：Karmada 使用 kubernetes 原生 API 定义用于联邦化的资源模板，使其易于与已采用 Kubernetes 的现有工具集成。
+- **分发策略**：Karmada 提供独立的 Propagation(placement) Policy API 来定义多集群调度和分发要求。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:10]: 这里 Propagation 用了`分发`，下文 Propagation 是不是也要相应修改的？

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 10]
```diff
@@ -1,54 +1,50 @@
...
+[Federation v2（也称为 Kubefed）](https://github.com/kubernetes-sigs/kubefed)的基础上开发的。
+Karmada 继承了很多两者的概念。例如：
 
-- **Resource template**: Karmada uses Kubernetes Native API definition for federated resource template, 
-  to make it easy to integrate with existing tools that already adopt Kubernetes.
-- **Propagation Policy**: Karmada offers a standalone Propagation(placement) Policy API to define multi-cluster 
-  scheduling and spreading requirements.
-- **Override Policy**: Karmada provides a standalone Override Policy API for specializing cluster relevant
-  configuration automation. 
+- **资源模版**：Karmada 使用 kubernetes 原生 API 定义用于联邦化的资源模板，使其易于与已采用 Kubernetes 的现有工具集成。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:9]:
```suggestion
- **资源模板**：Karmada 使用 kubernetes 原生 API 定义用于联邦化的资源模板，使其易于与已采用 Kubernetes 的现有工具集成。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md [hunk 11]
```diff
@@ -67,31 +63,30 @@ NAME      VERSION   MODE   READY   AGE
 member1   v1.20.7   Push   True    66s
 ```
 
-Kubefed manages clusters with `Push` mode, however Karmada supports both `Push` and `Pull` modes.
-Refer to [Overview of cluster mode](https://karmada.io/docs/userguide/clustermanager/cluster-registration) for 
-more details.
+Kubefed 管理着 `PUSH`模 式的集群，但是 Karmada 支持 `PUSH` 和 `PULL` 两种模式。
+请参阅 [集群模式概述](https://karmada.io/docs/userguide/clustermanager/cluster-registration) 。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:67]:
```suggestion
Kubefed 管理着 `PUSH` 模式的集群，但是 Karmada 支持 `PUSH` 和 `PULL` 两种模式。
请参阅[集群模式概述](https://karmada.io/docs/userguide/clustermanager/cluster-registration)。
```

**Review by windsonsea — COMMENTED** [review]
Thanks for your revising.

File: i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md (hunk 9 above)
Comment (justedennnnn) [reply i18n/zh/docusaurus-plugin-content-docs/current/administrator/migration/migration-from-kubefed.md:10]: Done

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Comment by SAMZONG** [comment]
I think it's all great.

/lgtm
/approve

Thanks for your contribution.

**Comment by justedennnnn** [comment]
> I think it's all great.
> 
> /lgtm /approve
> 
> Thanks for your contribution.

Thanks for your review.

**Comment by justedennnnn** [comment]
> @justedennnnn Can you help to resolve the conflicts, the typo in `docs/administrator/migration/migration-from-kubefed.md` has been resolved in #416.

Done

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**